internal/pkg/cli/svc_deploy.go

```diff
@@ -261,7 +262,10 @@ func (o *deploySvcOpts) Execute() error {
 				return err
 			}
 		}
-		contd, err := o.prompt.Confirm(continueDeploymentPrompt, "")
+		contd, err := o.skipDiffPrompt, nil
+		if !o.skipDiffPrompt {
+			contd, err = o.prompt.Confirm(continueDeploymentPrompt, "")
+		}
```

Comment:
I think it's okay to write ⬇️  here? Because if we got `--diff-yes`  that means `noDeploy` should render to `false` 🤔 
```
if o.skipDiffPrompt {
 return true, nil
}
```

Reply:
oh I wasn't sure because l.277 in the same method seems to do the actual deployment, so if I were to return it skip the deployment altogether I think 💭

Reply:
oh lol that's right